pkg/kudoctl/packages/package_test.go

```diff
@@ -19,17 +17,14 @@ import (
 )
 
 func TestReadFileSystemPackage(t *testing.T) {
-	// Set Kubernetes random seed for deterministic test results on the name
-	rand.Seed(1)
-
 	tests := []struct {
 		name         string
 		instanceName string
 		path         string
 		goldenFiles  string
 	}{
-		{"zookeeper", "zookeeper-xn8fg", "testdata/zk", "testdata/zk-crd-golden1"},
-		{"zookeeper", "zookeeper-txhzt", "testdata/zk.tgz", "testdata/zk-crd-golden2"},
+		{"zookeeper", "zookeeper", "testdata/zk", "testdata/zk-crd-golden1"},
+		{"zookeeper", "zookeeper", "testdata/zk.tgz", "testdata/zk-crd-golden2"},
```

Comment:
We should aim to keep unique instance names here. Otherwise we'll run into issue when multiple tests are run in parallel.